inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -200,6 +210,33 @@ def is_submission(obj, eng):
     return False
 
 
+@with_debug_logging
+def _get_journal_title(obj, eng):
```

Comment:
I would move this function in `get_journal_coverage`, or remove it completely, since it's not used anywhere else ;)

Reply:
I wouldn't. These nested functions are not really Pythonic (`Flat is better than nested.`) and are wasteful, as they are redefined on every function invocation.

Reply:
So which approach should I choose in the end?